### PR TITLE
Test

### PR DESCRIPTION
**Pull request by tranandh87**
@priya006 

Sample PR

### CODE REVIEW

File: WebContent/WEB-INF/web.xml [hunk 1]
```diff
@@ -8,5 +8,6 @@
     <welcome-file>default.html</welcome-file>
     <welcome-file>default.htm</welcome-file>
     <welcome-file>default.jsp</welcome-file>
+    <welcome-file>default123.jsp</welcome-file>
   </welcome-file-list>
-</web-app>
\ No newline at end of file
+</web-app>
```
Comment (cpsbottestuser) [inline WebContent/WEB-INF/web.xml:13]: Make it cleaner

Comment (tranandh87) [inline WebContent/WEB-INF/web.xml:13]: Done

Comment (tranandh87) [inline WebContent/WEB-INF/web.xml:13]: Done again.

**Comment by tranandh87** [comment]
good to merge

**Comment by tranandh87** [comment]
good to merge

**Comment by tranandh87** [comment]
Good

**Comment by tranandh87** [comment]
good again

**Comment by tranandh87** [comment]
good again

**Comment by tranandh87** [comment]
good 1

**Comment by tranandh87** [comment]
good 2

**Pull request closed without merging**